.nvmrc

```diff
@@ -1 +1 @@
-16.10.0
+16.13.0
```

Comment:
We've gone with 'lts/gallium' in other repos. It saves upping the version with each minor update.

Reply:
done